.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,7 +22,6 @@ Fixes #
 I have:
 
 - [ ] Read and followed Crossplane's [contribution process].
-- [ ] Run `make reviewable` to ensure this PR is ready for review.
```

Comment:
@checkaayush `make reviewable` is still a valid command, it just lives in the `build` submodule. You may have to run `make` once to initialize the `build` submodule.

Reply:
Got it. Thanks for the quick review. Have made the change.